polynote-spark/src/main/scala/polynote/kernel/util/PlainServerCommand.scala

```diff
@@ -0,0 +1,23 @@
+package polynote.kernel.util
+
+import java.nio.file.{Path, Paths}
+
+object PlainServerCommand {
+
+  def apply(
+    sparkConfig: Map[String, String],
+    mainClass: String = "polynote.server.SparkServer"
+  ): Seq[String] = {
+
+    val allDriverOptions =
```

Comment:
Is this necessary for this mode? These options will really be per-notebook in this case so they'll get passed to the remote kernel process.

Reply:
I was reusing this as a way to specify java options to polynote itself (e.g., if you wanted to specify `-Dpolynote.debug=true`). 

Maybe it would be better to add a new key for these options though (since `polynote.remote.kernel` isn't a spark option anyways). Maybe a top-level `server` key?